zone/npc.cpp

```diff
@@ -1112,31 +1111,30 @@ void NPC::SpawnGridNodeNPC(const glm::vec4 &position, int32 grid_number, int32 z
 		strcat(npc_type->name, "(Stacked)");
 	}
 
-	npc_type->current_hp       = 4000000;
-	npc_type->max_hp           = 4000000;
-	npc_type->race             = 2254;
-	npc_type->gender           = 2;
-	npc_type->class_           = 9;
-	npc_type->deity            = 1;
-	npc_type->level            = 200;
-	npc_type->npc_id           = 0;
-	npc_type->loottable_id     = 0;
-	npc_type->texture          = 1;
-	npc_type->light            = 1;
-	npc_type->size             = 1;
-	npc_type->runspeed         = 0;
-	npc_type->merchanttype     = 1;
-	npc_type->bodytype         = 1;
-	npc_type->show_name        = true;
-	npc_type->findable         = true;
+	npc_type->current_hp = 4000000;
+	npc_type->max_hp = 4000000;
+	npc_type->race = 2254;
+	npc_type->gender = 2;
+	npc_type->class_ = 9;
+	npc_type->deity = 1;
+	npc_type->level = 200;
+	npc_type->npc_id = 0;
+	npc_type->loottable_id = 0;
+	npc_type->texture = 1;
+	npc_type->light = 1;
+	npc_type->size = 1;
+	npc_type->runspeed = 0;
+	npc_type->merchanttype = 1;
+	npc_type->bodytype = 1;
+	npc_type->show_name = true;
+	npc_type->findable = true;
+	strn0cpy(npc_type->special_abilities, "24,1^35,1", 512);
```

Comment:
Would setting special abilities work after creating the NPC below so constants could be used? Or does this have a different effect? (I'm not super familiar with this area)
```cpp
npc->SetSpecialAbility(IMMUNE_AGGRO, 1);
npc->SetSpecialAbility(NO_HARM_FROM_CLIENT, 1);

Reply:
I don't think it has a different effect, I just assumed hard-coding would be fine enough for these grid nodes.